### PR TITLE
fix(embed): external resources styles

### PR DESCRIPTION
**Pull request by jyash97**
## What kind of change does this PR introduce?
Minor Style Fix

## What is the current behavior?
image

## What is the new behavior?
image

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Locally Tested

## Checklist

- [ ] Documentation
- [x] Testing 
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2958/commits/7d8e46c8e441152bc658abbaaf23af97a489fc85 is at https://pr2958.build.csb.dev/s/new.

**Comment by siddharthkp** [comment]
Thanks for taking the time @jyash97!

This is a step in the right direction, merging it.

I think you uncovered a bigger problem in how we show google fonts - might be worth parsing google fonts to show only the name of the font 🤔 (with some indication of what it is)

Screenshot 2019-11-05 at 11 02 24 AM

Link: https://pr2958.build.csb.dev/embed/djdm7

One possible solution: 

Screenshot 2019-11-05 at 11 05 03 AM

**Comment by siddharthkp** [comment]
@CompuIves tiny change, deploy whenever

**Comment by jyash97** [comment]
Hey @siddharthkp thanks fir the quick review 💯 

this sounds even better as on the app also we only show the font name. I thought of doing this but was not sure if this was something that could be merged. If you want I can help you with this :)

File: packages/app/src/embed/components/Sidebar/ExternalResources/elements.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const Container = styled.div(
 export const Row = styled.div(
   css({
     display: 'flex',
+    flexWrap: 'wrap',
```
Comment (siddharthkp) [inline packages/app/src/embed/components/Sidebar/ExternalResources/elements.js]: TIL!

**Comment by siddharthkp** [comment]
@jyash97 If you want I can help you with this :)

Go for it!

**Comment by jyash97** [comment]
Cool, I will create a PR for this soon :)

**Comment by jyash97** [comment]
Hey @siddharthkp based on our discussion I have setup the PR for parsing the google font. Link: #2990

**Pull request merged**